shell/platform/fuchsia/flutter/tests/integration/touch-input/touch-input-test.cc

```diff
@@ -405,19 +460,19 @@ TEST_P(FlutterEmbedTapTest, FlutterEmbedTap) {
   FML_LOG(INFO) << "Client launched";
 
   {
-    // Embedded child view takes up the left side of the screen
+    // Embedded child view takes up the center of the screen
     // Expect a response from the child view if we inject a tap there
-    InjectTap(-500, -500);
+    InjectTap(0, 0);
     RunLoopUntil([this] {
       return LastEventReceivedMatches(
-          /*expected_x=*/static_cast<float>(display_width() / 4.0f),
-          /*expected_y=*/static_cast<float>(display_height() / 4.0f),
+          /*expected_x=*/static_cast<float>(display_width() / 6.06f),
```

Comment:
Could you explain this math? It's not obvious to me why we expect the denominator here to be 6.06f. It would help me personally to know what the expected layout of the overlay and child are. What's the expected size and position for each?

Reply:
Sure, the embedded child view is located at the center of the parent and has a size of 422 x 264 (which is parent size * 0.33). The parent has size of 1280 x 800, which is the size of the screen. The overlay is positioned on top of and aligned to the top right corner of the embedded child view, and is the same size (422 x 264).

When injecting tap at (0, 0), the parent view and overlay will receive it at (640, 400). The embedded child view will receive it at (211.2, 132) - which is 640 * 0.33 and 400 * 0.33.

1280 / 211.2 and 800 / 132 = 6.06